scripts/validator-info

```diff
@@ -385,6 +423,18 @@ class ValidatorStats(BaseStats):
             )
             return None
 
+    @staticmethod
+    def get_enabled_state_via_supervisorctl():
+        ret = subprocess.check_output(
+            "supervisorctl status indy-node | awk '{print $2}'; exit 0",
+            stderr=subprocess.STDOUT, shell=True
+        )
+        ret = ret.decode().strip()
+        if ret == 'STOPPED':
+            return False
+        else:
+            return True
+
```

Comment:
As far as I understand there could be other cases of states for not enabled service.
https://github.com/Supervisor/supervisor/blob/b609b6738121a43765ee5cb09fade07d1a286381/supervisor/states.py#L14. Could you please take them into account or add logging if they are not expected instead of returning True?

Reply:
done